src/redux/configureStore.js

```diff
@@ -1,5 +1,6 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';// eslint-disable-line
```

Comment:
- [ ] Kindly remove eslint disabling here.

Reply:
eslint disable removed. Thanks for these suggestion, I will recheck my code better before submitting future projects.